docs/libcurl/opts/CURLOPT_SSL_OPTIONS.md

```diff
@@ -86,6 +86,16 @@ certificate that supports client authentication in the OS certificate store it
 could be a privacy violation and unexpected.
 (Added in 7.77.0)
 
+## CURLSSLOPT_EARLYDATA
+
+Tell libcurl to try sending application data as TLS1.3 early data. This option
+is only supported for OpenSSL, WolfSSL, MbedTLS and GnuTLS. This option works
```

Comment:
This should only list GnuTLS for now, right?

Reply:
Yes, leftover from base PR.